test/linalg/eigen.jl

```diff
@@ -105,6 +105,12 @@ end
     end
 end
 
+let a = rand(10, 10)
+    f = sort(eigfact(a))
+    @test a ≈ f[:vectors] * Diagonal(f[:values]) / f[:vectors]
+end
```

Comment:
Since the purpose is to sort, should we test that too? E.g. `@test issorted(...)`